.travis/install.sh

```diff
@@ -10,8 +10,8 @@ if [[ $TRAVIS_OS_NAME == 'osx' ]]; then
             sudo `dirname $0`/get-pip.py
             sudo pip install tox
             ;;
-        py36)
-            # for py36, use homebrew python, which comes with pip.
+        py38)
+            # for py38, use homebrew python, which comes with pip.
```

Comment:
I considered suggesting to pin the version of python here, but doing so in brew apparently means [using a specific commit in homebrew's github repo](https://www.dineshjoshi.com/2019/07/installing-specific-version-of-python-on-macos-using-home-brew/), which feels a little gross.

So I think what we have here instead is fine.

Reply:
Yeah we'll need to update it like yearly. I decided a future @clokep can solve that problem. 👍